boards/Dashboard/main.c

```diff
@@ -220,16 +287,16 @@ int main(void) {
                         CAN_IDT_THROTTLE_L,
                         CAN_IDM_single);
     CAN_wait_on_receive(1,
-                        CAN_IDT_BMS_MASTER,
-                        CAN_IDT_BMS_MASTER_L,
+                        0x19,
```

Comment:
We should probably have all of this stuff up to date in the CAN api at some point